aml-development/ozp-backend
# fix(image): fixed sql too many variables

AMLNG-824

**Description**     
As a PKI user, I get an error 500 while retrieving images

If a PKI user does not have access to the correct security marking, he/she will be receive a 500 error whenever a GET call is made on /api/image/

**Acceptance Criteria**   
Users sees a Permission Denied error
 
**How to test code**    
`git checkout master && make dev run` 
Goto `/api/image/1` as pmurt, you should see 'too many sql variables error'

`git checkout refactor_20180118`
Goto `/api/image/1` as pmurt, you should see Permission Denied Error

Ensure all unit tests pass
`make ptest`

**Please Review**     
@aml-development/ozp-backend-team    

**Checklist**
